src/index.ts

```diff
@@ -258,11 +261,7 @@ export class Client {
 
   public async sendRequest(request: AxiosRequestConfig, callback?: Callback): Promise<any> {
     try {
-      request.headers = request.headers || {};
```

Comment:
@MrRefactoring removing this, causes TypeError: Cannot read property 'Authorization' of undefined. 
So this line needs to be back:
```js
request.headers = request.headers || {};
request.headers.Authorization = request.headers.Authorization || getAuthentication(this.config, request);
```